tools/processresults_test.py

```diff
@@ -0,0 +1,304 @@
+"""Unit tests for processing CUES results from raw JSON to CSV
+
+Run on command line with: python processresults_test.py
+Should print 7 exceptions (these are expected output from ProcessResults)
+and pass 8 tests.
+"""
+
+import datetime
+import json
+import processresults
+import os
+import unittest
+
+class TestProcessResults(unittest.TestCase):
+  
+  def setUp(self):
+    self.mock_results = [
+        {
```

Comment:
Wouldn't it be better to put this into a plain .json file? You already seem to be creating the file below anyways (processresults_test_input.json).
Having a separate json file allows readers of the code to know what to expect as input as well. For example, I'm also not clear why _ParseJsonObjects can't just use json.loads, perhaps it has something to do with how the json is written into the file when a survey result is received.